praw/models/stylesheet.py

```diff
@@ -21,3 +24,18 @@ class Stylesheet(PRAWBase):
     ``stylesheet``          The contents of the stylesheet, as CSS.
     ======================= ===================================================
     """
+
+    def __eq__(self, other: _Stylesheet) -> bool:
+        """Check if two stylesheets are equal."""
+        if isinstance(other, self.__class__):
+            return (
+                self.images == other.images
+                and self.stylesheet == other.stylesheet
+            )
+        return super().__eq__(other)
+
+    def __repr__(self):
+        """Return the object's REPR."""
```

Comment:
As a very minor note, I believe "repr" is short for "representation" and isn't an acronym, so this should be styled as "repr" rather than "REPR".